debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -99,13 +100,18 @@ protected void initReplicationSlot() throws SQLException, InterruptedException {
         try {
             if (shouldCreateSlot) {
                 LOGGER.debug("Creating new replication slot '{}' for plugin '{}'", slotName, plugin);
+
                 // there's no info for this plugin and slot so create a new slot
-                pgConnection().getReplicationAPI()
```

Comment:
I think it makes sense to create a issue to support `temporary` in `createReplicationSlot()` API of [Postgresql JDBC Driver](https://github.com/pgjdbc/pgjdbc).

Reply:
Done: https://github.com/pgjdbc/pgjdbc/issues/1305

Reply:
Good idea!